modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -29,7 +29,7 @@ This notification is returned when there is a Cartesian product in the plan.
 .Notification details
 [cols="<1s,<4"]
 |===
-|Neo4j code
+|Neo4j  code
```

Comment:
Now, this makes it look like this double space in "Neo4j code" is deliberate. Is that some nifty Asciidoc thing doing some rendering magic or what the purpose? I am curious.

```suggestion
|Neo4j code
```